### PR TITLE
rollouts cli: use "default" value for namespace and minor cleanups

### PR DESCRIPTION
**Pull request by droot**
This PR addresses the review comments from the PR https://github.com/GoogleContainerTools/kpt/pull/3807

### CODE REVIEW

File: commands/alpha/rollouts/advance/advance.go [hunk 1]
```diff
@@ -65,7 +65,9 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	rolloutName := args[0]
 	waveName := args[1]
 
-	rollout, err := rlc.Get(r.ctx, rolloutName)
+	// TODO(droot): plumb the namespace value from commandline flags
```
Comment (mortent) [inline commands/alpha/rollouts/advance/advance.go:68]: The `kpt live` code already have functionality for this. Presumably we want to use the default value from the kubeconfig and allowing the user to override this with the `--namespace` flag.

**Review by mortent — APPROVED** [review]

**Comment by mortent** [comment]
Can we also update the PR title to be something a bit more descriptive? This will end up in the release notes, but it is not very informative.

**Comment by droot** [comment]
> Can we also update the PR title to be something a bit more descriptive? This will end up in the release notes, but it is not very informative.

Done.

File: commands/alpha/rollouts/advance/advance.go (hunk 1 above)
Comment (droot) [reply commands/alpha/rollouts/advance/advance.go:68]: Great, we can use whatever existing commands are using.

Comment (droot) [reply commands/alpha/rollouts/advance/advance.go:68]: Added a good first issue as well for this: https://github.com/GoogleContainerTools/kpt/issues/3814

**Pull request merged**